crates/forge/bin/cmd/test/mod.rs

```diff
@@ -181,7 +181,7 @@ impl TestArgs {
 
         let test_options: TestOptions = TestOptionsBuilder::default()
             .fuzz(config.clone().fuzz)
-            .invariant(config.invariant)
+            .invariant(config.clone().invariant)
```

Comment:
move clone after .invariant to clone only that object instead of the entire config

same above for fuzz